kvbc/src/categorization/kv_blockchain.cpp

```diff
@@ -18,6 +18,14 @@ namespace concord::kvbc::categorization {
 
 using ::bftEngine::bcst::computeBlockDigest;
 
+template <typename T>
+void nullopts(std::vector<std::optional<T>>& vec, std::size_t count) {
+  vec.clear();
```

Comment:
Maybe add a call to `vec.resize(count);` here to minimize allocations?

Reply:
Nice idea, will do!